### PR TITLE
Release 0.22.0

### PR DESCRIPTION
**Pull request by VilleMiekkoja**
## Description

v0.22.0 release

### CODE REVIEW

**Review by ronijaakkola — APPROVED** [review]
Looks good to go!

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+
+### Design kit
+#### Added
+- Date field and date picker
+#### Fixed
+- Incorrect text color in TextInput Disabled state
+
+### React Components
+#### Added
+- New Date Input component with Datepicker
```
Comment (minevala) [inline CHANGELOG.md:19]: I guess we could write this "date input with date picker" as in documentation. My bad, 👋 I used different format 😁

Comment (minevala) [reply CHANGELOG.md:19]: Otherwise it is good to go

**Review by laurakarhu — APPROVED** [review]
Ready to roll

**Pull request merged**